src/qililab/settings/runcard.py

```diff
@@ -248,12 +249,18 @@ def get_parameter(
 
     # Runcard class actual initialization
     name: str
-    device_id: int
     chip: Chip
     buses: list[Bus]  # This actually is a list[dict] until the post_init is called
     instruments: list[dict]
     instrument_controllers: list[dict]
     gates_settings: GatesSettings
+    device_id: int = field(default=None)
```

Comment:
Pyling complaining about:

```The dataclasses.field() specifier should only be used as the value of an assignment within a dataclass, or within the make_dataclass() function.```

Since we are not using any extra capability of the `field()` function, why not simple just do:

```suggestion
    device_id: int = None
```

Reply:
Also removing its import! 🙌

Reply:
Commited it in the last commit, myself to see if CodeQuality passes with this

(but I have not removed the import though, I leave that to you!)

Reply:
Yeah mb, I added it on the last moment